modules/editorial-metadata/lib/components/editorial-metadata-manager.js

```diff
@@ -131,6 +131,10 @@ export default function EditorialMetadataManager( {
 											<Text>
 												<i>{ eMetadataTerm.description }</i>
 											</Text>
+											<Text>
+												{ eMetadataTerm.type.charAt( 0 ).toUpperCase() +
```

Comment:
This took me a bit to figure out. Could you maybe assign this to a variable like `metadataTypeUppercased` and put that in the `<Text>` instead?

Reply:
Yip, can do

Reply:
Moved this to a function, and I fixed the discrepancy between what's shown here and in the type dropdown when creating or editing an editorial metadata field.